playroom/components/animation.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import Lottie from 'lottie-react';
+
+import type {LottieComponentProps} from 'lottie-react';
+import type {ExclusifyUnion} from '../../src/utils/utility-types';
+
+type Props = Omit<LottieComponentProps, 'animationData'> &
+    ExclusifyUnion<{animationData: unknown} | {animationUrl: string}>;
```

Comment:
The component allows all the same props as `<Lottie />`, and also `animationUrl` for the cases in which you want to download the JSON data from an url